packages/typescript/test/fixtures/nodenext-resolution/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "compilerOptions": {
-    "module": "nodenext"
+    "moduleResolution": "nodenext"
```

Comment:
@NotWoods with these tests previously passing using the (incorrect?) `module` property, would this constitute a breaking change?

Reply:
The tests are still passing, see [nodenext-module/tsconfig.json](https://github.com/rollup/plugins/pull/1453/files/8ad0cd59eff6de147ae086722c5adc56bb1a77d6#diff-18d9a02ea4854ea5ec06af4550b68244da312bc5cb96268c571dfbe215d69c69).
What changed is that `module` is only used if `moduleResolution´ is not set .